integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/tpcxbb/TpcxbbLikeSpark.scala

```diff
@@ -16,55 +16,78 @@
 
 package com.nvidia.spark.rapids.tests.tpcxbb
 
-import org.apache.spark.sql.{DataFrame, SparkSession}
```

Comment:
Need to add 2020 copyrights on this file.

Reply:
Looks like the Tpc*Bench files needed to have their copyrights updated (thanks for doing that, and sorry for missing it in my earlier review) but these Tpc*LikeSpark files still need updated copyrights.